docker-compose.yml

```diff
@@ -467,6 +467,8 @@ services:
 
   mail:
     image: ghcr.io/juliushaertl/nextcloud-dev-mailhog:latest
+    ports:
+      - 8025:8025
     environment:
       VIRTUAL_HOST: "mail${DOMAIN_SUFFIX}"
```

Comment:
Fine to expose it but it would also be reachable through the reverse proxy (e.g. http://mail.local if that is set in your /etc/hosts to 127.0.0.1)